data.sql

```diff
@@ -3,3 +3,20 @@
 INSERT INTO animals (name) VALUES ('Luna');
 INSERT INTO animals (name) VALUES ('Daisy');
 INSERT INTO animals (name) VALUES ('Charlie');
+
+INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,weight_kg)VALUES ('Agumon','2020-12-03',0,true,10.23);
+
+INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,weight_kg)VALUES ('Gabumon','2018-11-15',2,true,8);
+
+INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,weight_kg)VALUES ('Pikachu','2021-01-07',1,false,15.04);
+
+INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,weight_kg)VALUES ('Charmander','2020-02-08',0,false,11),
```

Comment:
Noted. Table succsessfully updated with below script:
UPDATE animals
SET weight_kg = '-11'
WHERE name = 'Charmander';

Reply:
- [ ] As mentioned by the previous reviewer, it is requested that the weight of `Charmander` should be `-11kg` and it would be nice to update the query to include `-11` as requested.